components/search-page-components/RatingFilter.jsx

```diff
@@ -1,30 +1,27 @@
 "use client";
 import React, { useState } from "react";
-import Box from "@mui/material/Box";
-import Rating from "@mui/material/Rating";
-import StarIcon from "@mui/icons-material/Star";
+import dynamic from "next/dynamic";
 
-function RatingFilter() {
+const Box = dynamic(() => import("@mui/icons-material/Box"));
+const Rating = dynamic(() => import("@mui/icons-material/Rating"));
```

Comment:
Please update the path to "@mui/material/Rating" from '@mui/icons-material/Rating'.